### ISSUE TITLE
M11 — Model training [epic]

### ISSUE THREAD

**Issue by chuanseng-ng** [chuanseng-ng/tcg-analyzer#310]
**Epic for milestone M11 — Model training** · Track: Data/ML · Spec §69/M11, §25, §27, §59, §68 · ADR 0011, ADR 0008

> **Decomposed 2026-09-11** into #311–#314 after a four-agent evaluation (code
> audit, data readiness, project priorities, adversarial review). **Data-first:**
> three issues are actionable now; #314 is `blocked` until a published dataset
> version holds the chosen company's issued grades on all three splits.

## Purpose

Take the grade models from ADR 0011's declared baselines to the **first trained
per-company model, registered and scored**.

The evaluation's verdict: **plan now; build the training code when there is
something to train on.**

- **The testing half is built.**
  - Both evaluation harnesses: `tcg-evaluate-condition` (#188) and
    `tcg-evaluate-grading` (#242).
  - Frozen dataset versions, and `tcg-record-grading-outcome` (#165).
  - The model registry (#189), which is still empty.
  - The injected-predictor seam (`PREDICTING_ADAPTERS`).
- **The training half cannot be validated yet.**
  - `grading_outcomes` holds **0 rows**, and `pokemon-condition-v0.2.0`'s test
    split is 2 cards.
  - ADR 0011's rule needs **≥ 16 test-split grades per company** before even a
    perfect record's 95% Wilson bound clears 0.80. That is about 112 graded
    cards for one company, or about 330 for all three.
  - No card has been submitted to a grader yet.
- **The binding constraint is graded cards, not code**
  (`docs/model-training-and-testing.md`). Grading turnaround is the longest
  step in the loop, and nothing tracks it. This milestone starts that clock.

## Deliverables — spec §69/M11

- a graded-submission plan: which company first, batch size, cost in SGD,
  turnaround
- the condition baseline recorded for `pokemon-condition-v0.2.0`
- ADR 0011's and ADR 0008's scheduled reviews
- one company's trained grade model, registered through
  `tcg-register-model-bundle` and scored by `tcg-evaluate-grading`

### Acceptance

- One company's trained bundle is registered and scored on a held-out test
  split, and beats the heuristic baseline's record.
- §27's 80% is claimed only where the 95% Wilson lower bound clears 0.80.

## Initial ML success criterion — spec §27 (verbatim)

> Initial target:
>
> > **At least 80% of predicted grades within ±1 actual grade on a properly held-out evaluation set.**
>
> This is a project target, not a guaranteed performance specification.
>
> The test set must be isolated from training and tuning.

## Deliverables checklist

- [ ] #311 — `research(datasets): plan the first graded-submission batch`
- [ ] #312 — `chore(ml/evaluation): record the condition evaluation for pokemon-condition-v0.2.0`
- [ ] #313 — `docs(adr): hold the scheduled ADR 0011 and ADR 0008 reviews`
- [ ] #314 — `feat(ml/grading): train and register the first per-company grade model` (`blocked` on graded cards)

## Planned order

```text
#311 submission plan ──► cards photographed, ingested, shipped ──► slabs return
   │                                                              │
   ├──► #313 ADR reviews (by 2026-11-22 / 2026-11-26)                │
   │                                                              ▼
#312 condition baseline ─────────────────────────────►  #314 first trained model
```

#312 has no dependency and can land first.

## What binds the decomposition

- **A trained model enters only by beating the baseline on the benchmark.** It
  comes in through the injected-predictor seam and #189's registry, behind the
  same interface (ADR 0011's re-entry triggers; the #188 binding).
- **Every bundle is born `experimental`.** Registration takes no `--status`.
- **Calibration is fitted on train and validation only.** The test split is
  scored, never tuned against.
- **One model per company, and no shared `ml/grading/common`** (ADR 0011).
  Never a universal condition-to-grade mapping.
- **A model wanting more data does not make a rejected source permitted**
  (ADR 0008). Risk R1 (artwork) stays open.
- **The 0.50 confidence gate and the 80% target do not move to meet the data.**
  Changing either is a new ADR.

## Deliberately not filed, and why

| Not filed | Why | Filed when |
| --- | --- | --- |
| An ML framework dependency, a training scaffold | CLAUDE.md requires benchmark-driven model choice, and there is nothing to benchmark on yet. A heavy dependency would bloat the isolated worker image for no gain. | Chosen inside #314 |
| Promotion, retirement and serving from the registry | ADR 0011 gives the promote-and-retire path to the first trained artifact | Part of #314's Definition of Done |
| A fitted calibrator | It needs train and validation grades, and enters behind a bumped `GRADE_EVALUATION_VERSION` | The first published version with train and validation outcomes |
| A learned condition analyzer | 19 defect markers cannot show a win over the baseline, and #208 bars a learned model without new evidence | The annotated corpus grows well past today's |
| Moving consented uploads (#148) into `tcg_corpus` | This belongs to M6, which is still open, not to M11. Those rows can never carry a grade. | The host database holds consented rows |
| A second and a third company's model | Each is its own issue under an unchanged ADR 0011 | #314 lands |
